frontend/public/reducers/connectToFlags.ts

```diff
@@ -0,0 +1,34 @@
+import { connect } from 'react-redux';
+import * as _ from 'lodash-es';
+import { FLAGS } from '@console/shared/src/constants';
+import { RootState } from '../redux';
+import { FeatureState } from './features';
+
+export const stateToFlagsObject = (state: FeatureState, desiredFlags: string[]): FlagsObject =>
+  desiredFlags.reduce((allFlags, f) => ({ ...allFlags, [f]: state.get(f) }), {} as FlagsObject);
+
+const stateToProps = (state: FeatureState, desiredFlags: string[]): WithFlagsProps => ({
+  flags: stateToFlagsObject(state, desiredFlags),
+});
+
+export type FlagsObject = { [key: string]: boolean };
```

Comment:
this was moved but not deleted from  `reducers/features.ts ` together with some others